libpod/container_internal_linux.go

```diff
@@ -1584,6 +1584,14 @@ func (c *Container) generatePasswd() (string, error) {
 	if _, err := os.Stat(passwdPath); err == nil {
 		return passwdPath, nil
 	}
+	// Check if container has a /etc/passwd - if it doesn't do nothing.
+	passwdPath, err := securejoin.SecureJoin(c.state.Mountpoint, "/etc/passwd")
+	if err != nil {
+		return "", errors.Wrapf(err, "error creating path to container /etc/passwd")
+	}
+	if _, err := os.Stat(passwdPath); err != nil && os.IsNotExist(err) {
```

Comment:
We should probbaly return an err if err != bil and it is not ENOEXIST.

Reply:
Good point